app/talk/lib/project-section.coffee

```diff
@@ -0,0 +1,3 @@
+# takes in a project id and returns a correctly formatted talk section for that project
+
+module?.exports = (projectId) -> "project-#{projectId}"
```

Comment:
I'd recommend passing in the project resources itself instead of its ID. The output is arbitrary, and the project is the input that matters. That its ID is used is just a detail.